README.md

```diff
@@ -84,6 +84,7 @@ Supported Kubernetes resources are `pod`, `replicationcontroller`, `service`, `d
  `--field-selector`          |           | Selector (field query) to filter on. If present, default to ".*" for the pod-query.
  `--include`, `-i`           | `[]`      | Log lines to include. (regular expression)
  `--init-containers`         | `true`    | Include or exclude init containers.
+ `--input-format`, `-I`      | `default` | Specify input format. Currently support: [default, zap-json]
```

Comment:
Since the name of the option that specifies the output format is `output`, it seems that the name of the option that specifies the input format should be `input`, but I am torn.

Reply:
I'm a little bit hesitant about `--input` since it's too broad. If one day someone would like to add `--input` option for whatever reason, that person would have hard time finding an appropriate name.

Reply:
I feel that the flag name `--input-format` is a bit confusing because its purpose is to change the output format. We can consider it a formatter for `json-zap`, so how about `--formatter`?

```
stern --formatter zap-json <QUERY>
```

Reply:
Yeah, I was afraid that `--input-format` might sound misleading, one of the option I had in mind is `--reformat` 😆 .

`--formatter` sounds good for me, @superbrothers what do you think?

Reply:
I've changed `--input-format` to `--formatter`.